sp/src/game/server/ez2/npc_flyingpredator.cpp

```diff
@@ -927,10 +948,21 @@ bool CNPC_FlyingPredator::ShouldGib( const CTakeDamageInfo &info )
 	if ( m_bIsBaby )
 		return true;
 
-	// Stukabats gib for any non-crush, non-blast damage greater than 15 (TODO: cvar?)
-	if ( info.GetDamage() > 15.0f || (info.GetDamageType() & (DMG_CRUSH | DMG_BLAST) ) )
```

Comment:
Why specifically was the > 15.0f condition removed?

Reply:
It was superseded by the checks below.